packages/react-day-picker/src/types/Modifiers.ts

```diff
@@ -1,18 +1,29 @@
 import { Matcher } from './Matchers';
 
-/** The map of modifiers with an array of matcher. */
+/** Map of modifiers and an array of matcher. */
 export type Modifiers = Record<Modifier, Matcher[]> &
   Record<InternalModifier, Matcher[]>;
 
-/** The modifiers that are used internally in DayPicker. */
-export type InternalModifier =
-  | 'selected'
-  | 'disabled'
-  | 'hidden'
-  | 'today'
-  | 'range_start'
-  | 'range_end'
-  | 'range_middle';
+/** The modifiers that are used internally by DayPicker. */
+export enum InternalModifier {
```

Comment:
New `InternalModifier` enum: I actually added it to be able to comment these values in the docs.